lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -125,6 +132,15 @@ export class PopupClient extends StandardInteractionClient {
 
             if (serverParams.accountId) {
                 this.logger.verbose("Account id found in hash, calling WAM for token");
+                // end measurement for server call with native brokering enabled
+                if (fetchNativeAccIdMeasurement) {
+                    fetchNativeAccIdMeasurement.endMeasurement({
+                        success: true,
+                        isNativeBroker: true
+                    });
+                    fetchNativeAccIdMeasurement.flushMeasurement();
```

Comment:
Do we need to discard this measurement if accountId is **not** in the response?

Reply:
I added it and then deleted since it is conditionally created for native. But you have a point, let me sync up with Jason on "flush" and "discard" definitions and will make the changes.